libs/kml/serdes_geojson.hpp

```diff
@@ -0,0 +1,115 @@
+#pragma once
+
+#include "kml/types.hpp"
+
+#include "glaze/json.hpp"
+
+#include "base/exception.hpp"
+
+namespace kml
+{
+namespace geojson
+{
+
+// Data structures
+
+struct GeoJsonGeometry
+{
+  std::string type;
+  std::vector<glz::json_t> coordinates;
+
+  bool operator==(GeoJsonGeometry const & data) const
+  {
+    return type == data.type;  // && coordinates == data.coordinates;
```

Comment:
`properties` field contains `glz::json_t` structure which doesn't support `==` operator. What's the best way to test for equality?